src/ReportPortal.Shared/Reporter/Statistics/IStatisticsCounter.cs

```diff
@@ -22,6 +22,11 @@ public interface IStatisticsCounter
         /// </summary>
         TimeSpan Avg { get; }
 
+        /// <summary>
+        /// Duration of last measured request.
```

Comment:
No needed here, this is just accumulated final stats.

Reply:
In this situation we need to duplicate logic of getting information about duration of request (if it's really needed for us). Right now it's implemented in base executor class and used by `IStatisticCounter`.